tags.tf

```diff
@@ -0,0 +1,34 @@
+## Copyright © 2020, Oracle and/or its affiliates. 
```

Comment:
There used to be an issue with tag creation that prevented the user from deploying in their own compartment and everything had to be deployed at the root (if I recall properly). Or maybe it required some specific permission. 
Is that still an issue? 
I don't want tags to add constraints to the ability to deploy anywhere.

Reply:
Tagging is a part of the OCI Architecture Standard. We cannot remove it.

Reply:
The question is: does this affect the ability to deploy in a specific compartment? I think that was the issue with creating tags. Because most users who use the lab won't be able to deploy on the root compartment, the deployment won't work for them if that's the case. If that's the case then we should make this tag creation optional, or check if the tag already exists, attempt to create it if the user has permission, but if not, not fail the stack.

Reply:
Tag Namespace and Tags are global objects, so I am using the home region OCI provider. Yes, we can create tag namespace & tags within a particular compartment. However, we used to add 120 seconds delay to wait for the tag distribution to all regions. Then we can build infra with tagging in every possible region. Tagging is unique for each RA, and each instantiation of the tag is equipped with a random suffix.